geometry_msgs/Vector3[] coordinates
string[] names
---
geometry_msgs/Vector3[] orderedCoordinates
string[] orderedNames